website/docs/docs/cloud/migration.md

```diff
@@ -0,0 +1,41 @@
+--- 
+title: "Multi-cell migration checklist"
+id: migration 
+description: "Prepare for account migration to AWS cell based architecture." 
+pagination_next: null
+pagination_prev: null
+---
+
+dbt Labs is in the process of migrating our U.S. based multi-tenant accounts to [AWS cell-based architecture](https://docs.aws.amazon.com/wellarchitected/latest/reducing-scope-of-impact-with-cell-based-architecture/what-is-a-cell-based-architecture.html), a critical component of the [AWS well-architected framework](https://aws.amazon.com/architecture/well-architected/?wa-lens-whitepapers.sort-by=item.additionalFields.sortDate&wa-lens-whitepapers.sort-order=desc&wa-guidance-whitepapers.sort-by=item.additionalFields.sortDate&wa-guidance-whitepapers.sort-order=desc). The benefits of the cell-based architecture will improve the performance, reliability, and security of your dbt Cloud environment, but there is some preparation required to ensure a successful migration.
+
+This document outlines the steps that you must take to prevent service disruptions before your environment is migrated over to the cell-based architecture. This will impact areas such as login, IP restrictions, and API access. 
+
+### What’s changing? Pre-migration checklist.
```

Comment:
suggs: 

- i generally defer to Merriam Webster as the de facto dictionary for American English https://www.merriam-webster.com/dictionary/premigration#:~:text=%3A%20existing%20or%20occurring%20before%20a,way%20to%20explaining%20group%20success%20%E2%80%A6 
- looks like 2 sentences vs a section title
- H2 vs H3. i've been taught to not skip headings (unless unnecessary to avoid)

```suggestion
## Premigration checklist
```

Reply:
Good catch on the H3, thank you. The definition linked has "pre-migration" as alternative usage and since that's how it's frequently used by tech articles (including [Microsoft](https://techcommunity.microsoft.com/t5/azure-database-for-postgresql/introducing-pre-migration-validations-for-single-server-to/ba-p/3963366)), I think we should keep the hyphen